### DISCUSSION TITLE
development server not starting

### DISCUSSION THREAD

**Discussion by iusegithub** [vercel/next.js#48277] [Help] [answered]
### Summary

OS :  macOS M1 Venture 13.3.1 
Node : 14.7.0
---

- `yarn dev` or `npm run dev` doesn't start development server. Ran into this while following the tutorial on https://nextjs.org/learn/basics/create-nextjs-app/setup

- `yarn dev` or `npm run dev` just runs without error but doesn't start the development server on browser. Opening http://localhost:3000/ from browser manually shows nothing either.

<img width="368" alt="Screenshot 2023-04-12 at 3 59 23 pm" src="https://user-images.githubusercontent.com/19864589/231364676-200534de-4083-43fe-be2f-e436e116a547.png">

- when I run `npx next dev` directly, I get the following error:

<img width="675" alt="Screenshot 2023-04-12 at 4 04 42 pm" src="https://user-images.githubusercontent.com/19864589/231365587-e0cd3d98-1590-44b5-9c46-3d7eded3a196.png">

- following is the output of `npx next info`

<img width="855" alt="Screenshot 2023-04-12 at 4 14 43 pm" src="https://user-images.githubusercontent.com/19864589/231367416-bde8f6fc-0b59-4a3e-a358-a8be84dcc320.png">

- have already tried removing `node_modules` and running `npm i` again

- have also run `node -p "process.arch"` (which outputs x64) but doesn't help either

**Comment by icyJoseph** [comment, marked as answer]
Have you tried to upgrade to Node 16+?

I tried to downgrade mine to 14, but I can't, because for some reason it fails to unpack it. 

I noticed I had 14.17.1 installed, but I can't even install packages with that version, as there's an eslint plugin that requires higher 14 versions.

**Reply by iusegithub** [reply]
Hey @icyJoseph , thanks for the response. 

I haven't upgraded to Node 16+ because I need to get this app to be developed in Node 14 runtime.

But you're right I was able to get it to work on a later Node (via NVM) on Node 14.21.1. Given there's issue on certain Node 14+ versions (i.e, not all work) would you suggest we ask team to update [this doc](https://nextjs.org/learn/basics/create-nextjs-app/setup#:~:text=If%20you%20don%E2%80%99t%20have%20Node.js%20installed%2C%20install%20it%20from%20here.%20You%E2%80%99ll%20need%20Node.js%20version%2010.13%20or%20later) since it's meant to be for a starter?

<img width="706" alt="Screen Shot 2023-04-13 at 9 42 03 am" src="https://user-images.githubusercontent.com/19864589/231609493-dd4c05ac-5ea1-4123-8292-d72c541dd7bd.png">

**Reply by icyJoseph** [reply]
That's odd.. I know the minimum version got updated when Next 13 was released, they probably forgot to update it there.

- Official docs: https://nextjs.org/docs/upgrading#v13-summary
- Blog post announcement: https://nextjs.org/blog/next-13#breaking-changes

> The minimum Node.js version has been bumped from 12.22.0 to 14.6.0, since 12.x has reached end-of-life.

I wouldn't call the learning material docs though, but you are right that should be updated. 

I've sent them a feedback note on the learning docs, and opened an issue on their next-learn repository, https://github.com/vercel/next-learn/issues/117

**Reply by iusegithub** [reply]
Thank you

**Comment by VinceCYLiao** [comment]
I had same issue as you when using Node.js version 14.17.0. After switching to 16.17.0, running 'yarn dev' properly started the development server.

**Reply by iusegithub** [reply]
Hey @VinceCYLiao , thanks for the response.

I haven't upgraded to Node 16+ because I need to get this app to be developed in Node 14 runtime.

But you're right I was able to get it to work on a later Node (via NVM) on Node 14.21.1. Given there's issue on certain Node 14+ versions (i.e, not all work) would you suggest we ask team to update [this doc](https://nextjs.org/learn/basics/create-nextjs-app/setup#:~:text=If%20you%20don%E2%80%99t%20have%20Node.js%20installed%2C%20install%20it%20from%20here.%20You%E2%80%99ll%20need%20Node.js%20version%2010.13%20or%20later) since it's meant to be for a starter?
